### PR TITLE
Rewording

### PR DESCRIPTION
**Pull request by sigoa**
the sentences are too long.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-Bitmessage is a collaborative project. You are welcome to submit pull requests 
-although if you plan to put a non-trivial amount of work into coding new
-features, it is recommended that you first solicit feedback on the DevTalk
-pseudo-mailing list:
-BM-2D9QKN4teYRvoq2fyzpiftPh9WP9qggtzh
+
+Bitmessage is a collaborative project. You are welcome to submit pull requests - although if you plan to put a non-trivial amount of work into coding new features, it is recommended that you first solicit feedback on the DevTalk pseudo-mailing-list ("subscribe"): http://bm6hsivrmdnxmw2f.onion/chan/DevTalk 
+
+Feel welcome to join chan "bitmessage", BM-2cWy7cvHoq3f1rYMerRJp8PT653jjSuEdY 
+which is on preview here: http://beamstat.com/chan/bitmessage
```
Comment (PeterSurda) [inline README.md]: This is a good point.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
 
 Development
 ----------
-Bitmessage is a collaborative project. You are welcome to submit pull requests 
-although if you plan to put a non-trivial amount of work into coding new
-features, it is recommended that you first solicit feedback on the DevTalk
-pseudo-mailing list:
-BM-2D9QKN4teYRvoq2fyzpiftPh9WP9qggtzh
+
+Bitmessage is a collaborative project. You are welcome to submit pull requests - although if you plan to put a non-trivial amount of work into coding new features, it is recommended that you first solicit feedback on the DevTalk pseudo-mailing-list ("subscribe"): http://bm6hsivrmdnxmw2f.onion/chan/DevTalk
```
Comment (PeterSurda) [inline README.md]: The DevTalk is gone. You can remove references to it. You can add reference to the contribution guidelines though (you can just copy&paste it from the wiki)

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-merlot
```
Comment (PeterSurda) [inline _config.yml]: don't include this file in the PR

File: build/notes.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+just a scratchpad.
```
Comment (PeterSurda) [inline build/notes.txt]: don't include this file in the PR

File: src/bitmessageqt/__init__.py [hunk 5]
```diff
@@ -3873,7 +3873,7 @@ def tableWidgetInboxItemClicked(self):
         else:
             data = self.getCurrentMessageId()
             if data != False:
-                message = "Error occurred: could not load message from disk."
+                message = "Error occurred: could not load message from disk." #### TODO: _translate( this ! happens frequently
```
Comment (PeterSurda) [inline src/bitmessageqt/__init__.py]: please put the TODO in the line before the message.

Comment (g1itch) [reply src/bitmessageqt/__init__.py]: Just drop this patch. I've already marked this string for translation and also completely rewritten the method.

**Comment by sigoa** [comment]
Okay I try to stash it .  :+1:

**Pull request closed without merging**